tests/utils.py

```diff
@@ -38,24 +35,36 @@ def get_obj_cfg(name, category, model, prim_type=PrimType.RIGID, scale=None, abi
     }
 
 def assert_test_scene():
+    objects_cfg = [
+        get_obj_cfg("breakfast_table", "breakfast_table", "skczfi"),
+        get_obj_cfg("bottom_cabinet", "bottom_cabinet", "immwzb"),
+        get_obj_cfg("dishtowel", "dishtowel", "dtfspn", prim_type=PrimType.CLOTH),
+        get_obj_cfg("carpet", "carpet", "ctclvd", prim_type=PrimType.CLOTH),
+        get_obj_cfg("bowl", "bowl", "ajzltc"),
+        get_obj_cfg("bagel", "bagel", "zlxkry", abilities=TEMP_RELATED_ABILITIES),
+        get_obj_cfg("cookable_dishtowel", "dishtowel", "dtfspn", prim_type=PrimType.CLOTH, abilities=TEMP_RELATED_ABILITIES),
+        get_obj_cfg("microwave", "microwave", "hjjxmi"),
+        get_obj_cfg("stove", "stove", "yhjzwg"),
+        get_obj_cfg("fridge", "fridge", "dszchb"),
+        get_obj_cfg("plywood", "plywood", "fkmkqa", abilities={"flammable": {}}),
+    ]
+
+    # Create the scene if there's any mismatch in objects
+    create_scene = False
     if og.sim.scene is None:
+        create_scene = True
+    else:
+        for obj_cfg in objects_cfg:
+            if og.sim.scene.object_registry("name", obj_cfg["name"]) is None:
+                create_scene = True
+                break
```

Comment:
hmm why is this needed?

Reply:
pytest naively crashes because after it runs all the tests from `test_envs.py`, it then starts to try to immediately run the tests from `test_object_states.py`, without actually destroying / clearing the sim. So when `assert_test_scene()` gets called, the scene already exists so it doesn't create the new scene, and all the tests crash because none of the actual test objects exist.

Reply:
reverted